### DISCUSSION TITLE
Can I reuse Cookie table in my privacy policy ?

### DISCUSSION THREAD

**Discussion by PeterWorakarn** [orestbida/cookieconsent#299] [General]
![image](https://user-images.githubusercontent.com/57410426/165509231-edd682cc-5a0a-446d-8344-d29dad2ed247.png)
I want to use cookie table in my privacy policy. Can I export via cc-data syntax?

**Comment by orestbida** [comment]
Hi @PeterWorakarn,

unfortunately, this is not possible. You could use JavaScript to grab the table inside the modal and append it to your page, but no style would be applied.
